packages/cosmic-swingset/gov-code.js

```diff
@@ -0,0 +1,7 @@
+/* global E, powers */
+const {
+  consume: { client },
+} = powers;
+
+console.info('hello from the governed code!');
+E(client).assignBundle([addr => ({ youAreGoverned: addr })]);
```

Comment:
move this under `/test/` maybe? Somehow clarify its purpose and that it's not part of the production code.